frontend_app_folder_advanced/test/container/FolderAdvanced.spec.js

```diff
@@ -45,25 +45,32 @@ describe('<FolderAdvanced />', () => {
   mockGetSystemContentTypes200(apiUrl, [{ label: 'folder' }, { label: 'html-document' }]).persist()
   mockGetFolder200(apiUrl, folder.workspace_id, folder.content_id, folder).persist()
 
+  const wrapper = shallow(<FolderAdvanced {...props} />)
+
   describe('TLM Handlers', () => {
     describe('eventType content folder', () => {
       describe('handleFolderChanged (same folder)', () => {
         const tlmData = {
+          author: {
+            avatar_url: null,
+            public_name: 'Global manager',
+            user_id: 1
+          },
           content: {
             ...folder,
             label: 'Hello, world'
           }
         }
 
-        const wrapper = shallow(<FolderAdvanced {...props} />)
+        // const wrapper = shallow(<FolderAdvanced {...props} />)
```

Comment:
if you don't use this line anymore, feel free to remove it

Reply:
Changed